Dockerfile

```diff
@@ -11,22 +11,17 @@ RUN apt-get update && \
         python3 \
         python-is-python3 \
         python3-pip \
+        python3.10-venv \
         libgl1 \
         libgl1-mesa-glx \ 
         libglib2.0-0 \
     && rm -rf /var/lib/apt/lists/*
-
+    
 ADD requirements.txt .
 
-RUN pip install torch==2.0.0+cu117 torchvision==0.15.1+cu117 torchaudio==2.0.1 --index-url https://download.pytorch.org/whl/cu117 && \

```

Comment:
we may only need to switch to `pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cu118`. Cuda 12 requires higher version of driver, which may be a concern.

Reply:
BTW we can specify pypi source from requirements.txt, so we can keep `RUN pip install -r requirements.txt` but instead update the requirements.txt with proper pypi source.